src/common/src/config.rs

```diff
@@ -1307,13 +1312,17 @@ pub mod default {
         }
 
         pub fn shared_buffer_capacity_mb() -> usize {
-            1024
+            4096
```

Comment:
We use total memory to decide this configuration, you shall change it in other place and make sure that `shared_buffer_min_batch_flush_size_mb` is larger than default `shared_buffer_capacity_mb`

Reply:
I just realized that this `default::shared_buffer_capacity_mb` is only used in (https://github.com/risingwavelabs/risingwave/blob/4e64389601dfb14c566f2d4dc52fc2a753266c49/src/common/src/config.rs#L2059). That means it is only relevant to test. Let's keep the original value instead of changing it. We may find a way to deprecate it as well (in a separate PR)

Reply:
> We use total memory to decide this configuration, you shall change it in other place and make sure that shared_buffer_min_batch_flush_size_mb is larger than default shared_buffer_capacity_mb

We can do the check here: https://github.com/risingwavelabs/risingwave/blob/4e64389601dfb14c566f2d4dc52fc2a753266c49/src/compute/src/memory/config.rs#L71